src/asciidoc/docs/distribute/app-store.adoc

```diff
@@ -0,0 +1,86 @@
+= Listing in the App Store
+:page-slug: app-store
+:page-description: An introduction to the Koji App Store and how to list your Koji app.
+:includespath: ../_includes
+
+After you create, test, and move your app to production, you can share its production URL or list the app in the Koji App Store.
+
+The https://withkoji.com/apps[Koji App Store] is the marketplace for all Koji apps.
+When users edit their Koji profile, they can choose to add an app from the App Store.
+
+By listing in the Koji App Store, you can take advantage of the traffic in the App Store to get exposure for your app.
+Koji also announces noteworthy apps in social media posts.
+
+This topic shows how to create a Koji App Store listing after you move your app to production.
+
+NOTE: You can have multiple listings for the same app.
+
+== Creating an App Store listing
+
+You need the following for your App Store listing:
```

Comment:
Could this list be in a separate section? E.g., “Creating assets,” “Gathering assets” or something like that.

Reply:
I don't think we should call these assets. They're really only required for the App Store listing. 

When I created the listings, I found that I needed to create the images while I was filling out the form. If I lost network connection in the middle, or if my computer crashed, I'd have to start filling the form all over again. So it's just a better workflow to create the images first (and know what the required dimensions are) before starting on the form.

Reply:
Yes, these are definitely marketing assets for a listing. It’s worthwhile to call them out in a separate section; we might even provide guidance in the future on how to make the best marketing assets. It will also make the “Creating a listing” procedure easier to read. Actually, your observations about creating/gathering those assets before starting the listing would make a good tip/intro to the procedure.

Reply:
Separated the list into its own section and added a tip to prepare them first.

Reply:
Looks great!